Emitron/Emitron/UI/Shared/Content Detail/TextListItemView.swift

```diff
@@ -69,11 +70,16 @@ struct TextListItemView: View {
           Spacer()
             
           if canDownload {
+            VStack {
+              Spacer()
```

Comment:
Nit - the indentation for the code below this seems to be off? Is that right or just GH?